src/llm_utils.py

```diff
@@ -54,18 +54,13 @@ def analyze_code_changes(diff_content: str) -> List[Dict]:
         model="gemini-2.0-flash", contents=prompt
     )   
 
-    #print(response.text)
```

Comment:
Remove the print statement as it is likely used for debugging.

```suggestion
Remove print(response.text)
```